libs/cardano-ledger-conformance/test/Test/Cardano/Ledger/Conformance/Imp.hs

```diff
@@ -0,0 +1,144 @@
+  , ForAllExecSpecRep NFData ConwayFn "LEDGER" era
+  , ForAllExecSpecRep ToExpr ConwayFn "LEDGER" era
+  , NFData (SpecRep (PredicateFailure (EraRule "LEDGER" era)))
+  , ToExpr (SpecRep (PredicateFailure (EraRule "LEDGER" era)))
+  , Eq (SpecRep (PredicateFailure (EraRule "LEDGER" era)))
+  , FixupSpecRep (SpecRep (PredicateFailure (EraRule "LEDGER" era)))
+  , EncCBOR (ExecContext ConwayFn "LEDGER" era)
+  , EncCBOR (Environment (EraRule "LEDGER" era))
+  , EncCBOR (State (EraRule "LEDGER" era))
+  , ToExpr (ExecContext ConwayFn "LEDGER" era)
+  , HasCallStack
+  ) =>
+  Either
+    (NonEmpty (PredicateFailure (EraRule "LEDGER" era)))
+    (State (EraRule "LEDGER" era), [Event (EraRule "LEDGER" era)]) ->
+  ExecEnvironment ConwayFn "LEDGER" era ->
+  ExecState ConwayFn "LEDGER" era ->
+  ExecSignal ConwayFn "LEDGER" era ->
+  ImpM (LedgerSpec era) ()
+testImpConformance impRuleResult env state signal = do
+  logDoc $
+    mconcat
+      [ "\n----- LedgerEnv:\n"
+      , ansiExpr env
+      , "\n----- LedgerState:\n"
+      , ansiExpr state
+      , "\n----- Transaction:\n"
+      , ansiExpr signal
+      ]
+  slot <- use impLastTickG
+  let ctx :: ConwayLedgerExecContext era =
+        ConwayLedgerExecContext
+          { clecPolicyHash = SNothing
+          , clecEnactState = def
+          , clecUtxoExecContext =
+              UtxoExecContext
+                { uecTx = signal
+                , uecUTxO = state ^. lsUTxOStateL . utxosUtxoL
+                , uecUtxoEnv =
+                    UtxoEnv
+                      { ueSlot = slot
+                      , uePParams = state ^. lsUTxOStateL . utxosGovStateL . curPParamsGovStateL
+                      , ueCertState = state ^. lsCertStateL
+                      }
+                }
+          }
+  logDoc $ "\n----- ConwayLedgerExecContext:\n" <> ansiExpr ctx
+
+  (specEnv, specState, specSignal) <-
+    impAnn "Translating the inputs" $
+      translateInputs @ConwayFn @"LEDGER" @era env state signal ctx
+  logDoc $
+    mconcat
+      [ "\n----- spec LedgerEnv:\n"
+      , ansiExpr specEnv
+      , "\n----- spec LedgerState:\n"
+      , ansiExpr specState
+      , "\n----- spec Transaction:\n"
+      , ansiExpr specSignal
+      ]
+
+  agdaResponse <-
+    fmap (bimap (fixup <$>) fixup) $
+      impAnn "Deep evaluating Agda output" $
+        evaluateDeep $
+          runAgdaRule @ConwayFn @"LEDGER" @era specEnv specState specSignal
+
+  impResponse <-
+    impAnn "Translating implementation values to SpecRep" $
+      expectRightExpr $
+        runSpecTransM ctx $
+          bimapM
+            (traverse toTestRep)
+            (toTestRep . inject @_ @(ExecState ConwayFn "LEDGER" era) . fst)
+            impRuleResult
+
+  checkConformance @"LEDGER" @era @ConwayFn
+    ctx
+    (inject env)
+    (inject state)
+    (inject signal)
+    impResponse
+    agdaResponse
+
+spec :: Spec
+spec =
+  withImpInit @(LedgerSpec Conway) $ do
+    xdescribe "Tx conformance"
```

Comment:
So they are not passing yet?  The plan is to merge and fix them later?

Reply:
Yeah, that's how I usually did with conformance tests also, this PR just sets up the infrastructure for running the tests. It's hard to predict how long it might take to fix a conformance failure + there might be other issues cropping up after one is fixed. I'd like to have this PR merged so I can add a similar hook to `passTick` without having to worry about rebasing.

Reply:
Makes sense, thanks for explaining.